hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmVolumeArgs.java

```diff
@@ -82,8 +94,36 @@ private OmVolumeArgs(String adminName, String ownerName, String volume,
     this.modificationTime = modificationTime;
     this.objectID = objectID;
     this.updateID = updateID;
+    this.refCount = refCount;
+  }
+
+  public long getRefCount() {
+    Preconditions.checkState(refCount >= 0, "refCount should not be negative");
```

Comment:
This check should be moved to the decrement operation, where the refCount value actually changes. Otherwise we will not know where the actual error occurred.